rules/okta_rules/okta_phishing_attempt_blocked_by_fastpass.py

```diff
@@ -0,0 +1,20 @@
+from panther_base_helpers import deep_get, okta_alert_context
+
+
+def rule(event):
+    return (
```

Comment:
Same comment re: default for `get` and both `deep_get` calls.

Reply:
added default strings for human readable title, not necessary for boolean logic imo